chapters/core/setup/index.rst

```diff
@@ -13,3 +13,5 @@ Below you will find the setup guides for various, selected environments.
 
    ../../install/index
 
+.. note:: You can use Jina without even installing it via `Google Colab
```

Comment:
`You can use Jina with Google Colab with runtime configuration of your choice, without any installation hassles` might be a better hint at using colabs

Reply:
I copied what Tensorflow says, I think short and sweeter is better? 

Cc @alexcg1 wdut?

Reply:
It's a minor thing, we can let it be as is, idea is to elaborate what `without even installing` would entail and that Colab offers runtime configs (which is implied for colab users anyway).